syntaxes/idris.tmLanguage

```diff
@@ -121,6 +121,12 @@
 			</array>
 		</dict>
 		<dict>
+			<key>match</key>
+			<string>(?&lt;!\w)\'\w++(?!')</string>
```

Comment:
Sorry, I don't get the point, I think currently `'abcdef'` just works like a charm

![image](https://cloud.githubusercontent.com/assets/6234553/23202409/c9d0ae58-f918-11e6-9e25-1beb16fc52e2.png)